diffkemp/diffkemp.py

```diff
@@ -580,3 +590,98 @@ def text_indent(text, width):
             if not output_dir:
                 output.write(text_indent("}}}\n", indent - 2))
             output.write("\n")
+
+
+def view(args):
+    """
+    View the compare differences. Prepares files for the visualisation
+    and runs viewer.
+    """
+    # Load yaml describing results
+    YAML_FILE_NAME = "diffkemp-out.yaml"
+    yaml_path = os.path.join(args.compare_output_dir, YAML_FILE_NAME)
+    if not os.path.exists(yaml_path):
+        sys.stderr.write(
+            "ERROR: compare output is missing file " + f"'{YAML_FILE_NAME}'\n")
+        sys.exit(errno.EINVAL)
+    with open(yaml_path, "r") as file:
+        yaml_result = yaml.safe_load(file)
+
+    VIEW_DIRECTORY = os.path.join(os.path.dirname(__file__), "../view")
+
+    if args.devel:
+        PUBLIC_DIRECTORY = os.path.join(VIEW_DIRECTORY, "public")
+    else:
+        PUBLIC_DIRECTORY = os.path.join(VIEW_DIRECTORY, "build")
+
+    # Preparing source directory
+    SOURCE_DIRECTORY = os.path.join(PUBLIC_DIRECTORY, "src")
+    if os.path.isdir(SOURCE_DIRECTORY):
+        shutil.rmtree(SOURCE_DIRECTORY)
+    os.mkdir(SOURCE_DIRECTORY)
```

Comment:
Without the `--devel` option, this crashes unless the directory `{ROOT_DIFFKEMP_DIR}/view/build` already exists.
```suggestion
    os.makedirs(SOURCE_DIRECTORY, exist_ok=True)
```

Reply:
Oh, my bad, the directory is created by CMake.